### ISSUE TITLE
feature: maintain pin tree selection across reloads

### ISSUE THREAD

**Issue by born2net** [zalmoxisus/redux-devtools-extension#305]
Feature: maintain pin tree selection across reloads
As far as I can tell it is not possible now?!

![pin](https://cloud.githubusercontent.com/assets/5090936/22765682/8a833ef2-ee26-11e6-9e7f-7212c8dc7307.PNG)

TX

Sean

**Comment by zalmoxisus** [comment]
Yes, that's nice to have, together with [persisting tabs selections](https://github.com/zalmoxisus/redux-devtools-extension/issues/303). The problem however is that if on reloading the subobject is not present, we'll get `undefined` and not always it's obvious why. So, we'll also need a strategy for autoreseting the tree selection.

**Comment by born2net** [comment]
that would be an amazing feature as I find myself having to keep drilling down the tree into deeply nested nodes on every rebuild
tx as always

Sean

**Comment by wulftone** [comment]
Is this feature available now?

**Comment by pcbailey** [comment]
I just recently started using Redux dev tools, but have known about it for a long time. I was really enjoying the ability to view the state tree and inspect it at will without using the usual debugger and console.log tricks, but I was shocked to discover that this feature is missing and that I have to manually drill down every time I refresh the page. I don't know why this feature was abandoned, but please either reconsider adding it, or at the very least explain here why it's not possible or isn't a priority and close the issue.
